aea/cli/add.py

```diff
@@ -159,6 +159,32 @@ def add_item(ctx: Context, item_type: str, item_public_id: PublicId) -> None:
     click.echo(f"Successfully added {item_type} '{item_config.public_id}'.")
 
 
+def find_item(
```

Comment:
```suggestion
def fetch_item(
```

Reply:
We're actually fetching the package from the provided registry type

Reply:
done